test/integration/pkg/generated/openapi_generated.go

```
@@ -103,6 +104,12 @@ func schema_test_integration_testdata_custom_FooV3OneOf(ref common.ReferenceCall
 	})
 }
 
+func schema_test_integration_testdata_custom_FooV3Raw(ref common.ReferenceCallback) common.OpenAPIDefinition {
+	schema := spec.Schema{}
+	_ = schema.UnmarshalJSON(custom.FooV3Raw{}.OpenAPISchemaJSON())
```

ignoring the error is not great... panic seems worse